07-sampling.Rmd

```diff
@@ -73,128 +70,111 @@ percent_red_chr <- prop_red %>% percent(accuracy = 0.1)
-
-Take a look at the bowl in Figure \@ref(fig:sampling-exercise-1). It has a certain number of red and a certain number of white balls all of equal size. `r if_else(is_latex_output(), '(Note that in this printed version of the book "red" corresponds to the darker-colored balls, and "white" corresponds to the lighter-colored balls. We kept the reference to "red" and "white" throughout this book since those are the actual colors of the balls as seen in the background of the image on our book\'s [cover](https://moderndive.com/images/logos/book_cover.png).)', '')` Furthermore, it appears the bowl has been mixed beforehand, as there does not seem to be any coherent pattern to the spatial distribution of the red and white balls.
-
-Let's now ask ourselves, what proportion of this bowl's balls are red?
+Take a look at the bowl in Figure \@ref(fig:sampling-exercise-1). It has a certain number of red balls and a certain number of white balls all of equal size. `r if_else(is_latex_output(), '(Note that in this printed version of the book "red" corresponds to the darker-colored balls, and "white" corresponds to the lighter-colored balls. We kept the reference to "red" and "white" throughout this book since those are the actual colors of the balls as seen in the background of the image on our book\'s [cover](https://moderndive.com/images/logos/book_cover.png).)', '')` The balls in the bowl have been mixed beforehand, as there does not seem to be any coherent pattern to the spatial distribution of the red and white balls. Our task is to determine, at least with some level of precision, the proportion of red balls in the bowl. 
 
 ```{r sampling-exercise-1, echo=FALSE, fig.cap="A bowl with red and white balls.", purl=FALSE, out.width = "95%", purl=FALSE}
 include_graphics("images/sampling/balls/sampling_bowl_1.jpg")
 ```
 
-One way to answer this question would be to perform an exhaustive count: remove each ball individually, count the number of red balls and the number of white balls, and divide the number of red balls by the total number of balls. However, this would be a long and tedious process. 
+One way to do this is to perform an exhaustive count: remove each ball individually, count the number of red balls and divide the number of red balls by the total number of balls. Doing this carefully can determine the exact proportion of red balls in the bowl but it requires a long and tedious process.
 
 
-### Using the shovel once 
 
-Instead of performing an exhaustive count, let's insert a shovel into the bowl as seen in Figure \@ref(fig:sampling-exercise-2). Using the shovel, let's remove $5 \cdot 10 = 50$ balls, as seen in Figure \@ref(fig:sampling-exercise-3).
+### One sample
+
+Instead of performing an exhaustive count, we take a sample of balls. We do this by inserting a shovel into the bowl as shown in Figure \@ref(fig:sampling-exercise-2) and taking a sample of $5 \cdot 10 = 50$ balls as shown in Figure \@ref(fig:sampling-exercise-3).
```

Comment:
We haven't used the word "sample" here yet intentionally. Since that's an important concept, we'd rather build on the ideas and then introduce terminology later.